pyabc/populationstrategy.py

```diff
@@ -160,26 +166,34 @@ class AdaptivePopulationSize(PopulationStrategy):
             https://doi.org/10.1007/978-3-319-67471-1_8.
     """
 
-    def __init__(self, start_nr_particles, mean_cv=0.05,
-                 *,
-                 max_population_size=float("inf"),
-                 min_population_size=10,
-                 nr_samples_per_parameter=1,
-                 n_bootstrap=10):
-        super().__init__(start_nr_particles,
-                         nr_samples_per_parameter=nr_samples_per_parameter)
+    def __init__(self,
+                 start_nr_particles,
+                 mean_cv=0.05,
```

Comment:
space?
```suggestion
                 mean_cv = 0.05,
```

Reply:
right. not possible in python by flake8 standard unless also a type hint is given. but I added one now (that was missing), so now it's better.